### PR TITLE
feat: Add link to user's open-sauced goals

### LINKED ISSUE #948: Feature: add clickable link to the goals repo on the first-time setup
### Is your feature request related to a problem? Please describe.

After installing the Open Sauced GitHub application and creating the "open-sauced-goals" repository a link to it could be clickable before granting rights to the app.

Additional context can help what is happening after clicking the install button.

### Describe the solution you'd like

Add an anchor to open the GitHub repository in a new tab.

Add some additional description instead or before redirecting to the goals repository.

### Additional context

https://clips.twitch.tv/TenaciousKitschyDonkeyOSkomodo-SaSLo_p5485mkNTm

https://clips.twitch.tv/HealthyHealthyNewtPunchTrees-AivvMBrYgJPH3U7W

### PR DESCRIPTION
**Pull request by mtfoley**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [x] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description
#948 by adding a link to user's newly created open-sauced-goals repo. As mentioned in issue commentary, redirect to this repo after installation steps may require passing state to app's OAuth flow along with configuration changes to the open-sauced GitHub app itself.

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 docs
- [x] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

check

### CODE REVIEW

File: src/components/CreateGoals.js [hunk 1]
```diff
@@ -15,23 +15,28 @@ function CreateApp() {
...
       <small>
         <em>You own all your data saved while saucin.</em>
       </small>
     </React.Fragment>
   );
 }
 
-function InstallApp() {
+function InstallApp({user}) {
+  const repoUrl = "https://github.com/" + user.login + "/open-sauced-goals";
```
Comment (filiptronicek) [inline src/components/CreateGoals.js:27]: Maybe we could use [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for this string

**Review by filiptronicek — CHANGES_REQUESTED** [review]

File: src/components/CreateGoals.js (hunk 1 above)
Comment (mtfoley) [reply src/components/CreateGoals.js]: Clearly showing my age here LOL.

**Review by filiptronicek — APPROVED** [review]
Looks good to me! ✅

File: src/components/CreateGoals.js [hunk 2]
```diff
@@ -15,23 +15,28 @@ function CreateApp() {
...
       <small>
         <em>You own all your data saved while saucin.</em>
       </small>
     </React.Fragment>
   );
 }
 
-function InstallApp() {
+function InstallApp({user}) {
+  const repoUrl = `https://github.com/${user.login}/open-sauced-goals`;
```
Comment (bdougie) [inline src/components/CreateGoals.js:27]: This should link to open-sauced/goals-template instead

**Review by bdougie — CHANGES_REQUESTED** [review]
Just this one update and I think we can merge.

File: src/components/CreateGoals.js (hunk 2 above)
Comment (mtfoley) [reply src/components/CreateGoals.js:27]: So we don't need the user context then?

Comment (bdougie) [reply src/components/CreateGoals.js:27]: Wow, reviewing this on stream had me miss-read this. I event tested it wrong. This is good to go as is. The context of the user is needed.

**Review by bdougie — APPROVED** [review]
Sorry for the consufion. This LGTM

**Pull request merged**